load-test/scripts/configuration.js

```diff
@@ -60,8 +60,7 @@ export function getTestOptions(maxUserGoal, highLoadTimeMinutes) {
       exceptions: ["count == 0"],
       errors: ["count == 0"],
       timeouts: ["count == 0"],
-      total_session_time: ["p(95) < 5000"],
-      dropped_iterations: ["count == 0"]
+      total_session_time: ["p(95) < 5000"]
```

Comment:
(for my understanding) why are we removing this?

Reply:
Oops meant to add a comment explaining this! I had previously changed the test [executor](https://k6.io/docs/using-k6/scenarios/executors/) from per VU iterations to constant VUs, which meant a change from expecting x number of VUs to do y iterations in z minutes, to just having x VUs do as many requests as possible in z minutes. This meant dropped_iterations was no longer relevant, as that was reporting the iterations some VUs didn't get to execute in the time limit. We no longer have an expected number of iterations per VU so that number is always 0.